recipes-bsp/bootscripts/bootscript/boot.scr

```diff
@@ -11,7 +11,7 @@ elif test "${devtype}" = "ubi"; then
 	echo "Loading kernel from UBIFS..."
 	if ubifsload $fdt_addr /boot/system.dtb && ubifsload $kernel_addr /boot/uImage; then
 		echo "Load from UBIFS succeeded"
-		setenv bootargs "console=ttyPS0,115200 ubi.mtd=3 root=ubi0:qspi-rootfs rootfstype=ubifs rw rootwait"
+		setenv bootargs $bootargs console=ttyPS0,115200 ubi.mtd=3 root=ubi0:qspi-rootfs rootfstype=ubifs rw rootwait
```

Comment:
Die "console" mag hier dan ook weg denk ik?

Reply:
de console staat ook bij de mmc boot option voor de 7000 series. Vandaar dat ik hem hier heb laten staan. Ik kan hem ook een niveau omhoog gooien, maar misschien kan hij er idd. helemaal uit

Reply:
the console seems to be needed, I refactored this pull request